planb/remote_command.py

```diff
@@ -0,0 +1,61 @@
+import click
+import subprocess
+from .common import boto_client, list_instances
+
+
+def quoted(command: str) -> str:
+    return '"{}"'.format(
+        command.replace("\\", "\\\\").replace("\"", "\\\"").replace("$", "\\$")
+    )
+
+
+def run_ssh(
+        command: list, cluster_name: str, region: str, odd_host: str, piu: str,
+        echo: bool, no_prompt: bool, no_wait: bool, ip_label: bool):
+
+    ec2 = boto_client('ec2', region)
+    ips = [i['PrivateIpAddress']
+           for i in list_instances(ec2, cluster_name)
+           if 'PrivateIpAddress' in i]
```

Comment:
This pattern for `alive` instance seems to be repeatedly used. Function?

Reply:
It is actually made obsolete by this: https://github.com/zalando-stups/planb-cassandra/pull/210/files#diff-6e142b6ca2bb523496f5843fdf0e84a2R103 (though a PR is unrelated)